BuildingSync.xsd

```diff
@@ -796,6 +796,18 @@
           </xs:simpleContent>
         </xs:complexType>
       </xs:element>
+      <xs:element name="TotalExteriorRoofArea" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>Total roof area exposed to the elements. (ft2)</xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+          <xs:simpleContent>
+            <xs:extension base="xs:decimal">
```

Comment:
maybe make this as a non negative decimal type (we might need to add this as a base type). see the `BoundedDecimalZeroToOne` for something similar.